testsuite/forge/src/interface/admin.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{CoreContext, Test};
+use crate::Result;
+use diem_sdk::{
+    client::BlockingClient,
+    types::{chain_id::ChainId, LocalAccount},
+};
+
+#[derive(Debug)]
+pub struct AdminContext<'t> {
+    core: CoreContext,
+
+    admin_info: AdminInfo<'t>,
```

Comment:
I know this is a draft, but it would definitely be helpful to know what each of these represent e.g. `Admin`, `Network`, `Public` in some docs at the top.